### PR TITLE
Configurable time display format

### PR DESCRIPTION
**Pull request by louyihua**
Provide a setting option called `TIME_FORMAT` to provide such functionality.

This format string uses the PHP date() style, which is described at http://www.php.net/date

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @louyihua! I've created [OSPR-936](https://openedx.atlassian.net/browse/OSPR-936) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).
We can't start reviewing your pull request until you've   added yourself to the [AUTHORS](https://github.com/edx/edx-analytics-dashboard/blob/master/AUTHORS) file . Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

File: analytics_dashboard/settings/base.py [hunk 1]
```diff
@@ -321,8 +321,9 @@
 # used to construct shortcusts to view/edit courses in Studio
 CMS_COURSE_SHORTCUT_BASE_URL = None
 
-# Used to determine how dates are displayed in templates
+# Used to determine how dates and time are displayed in templates
```
Comment (sarina) [inline analytics_dashboard/settings/base.py:324]: It would be good to extend the comments here to explain what F, d, Y, g, i, A stand for.

Comment (sarina) [inline analytics_dashboard/settings/base.py:324]: Or, if there's documentation on all accepted abbreviations, perhaps just link to taht.

**Comment by sarina** [comment]
This makes sense to me, thanks @louyihua - it may be a bit of time though before the Analytics team has time to review this, given their current volume of PRs and upcoming US holidays.

File: analytics_dashboard/settings/base.py [hunk 2]
```diff
@@ -321,8 +321,10 @@
...
+# Uses the PHP's date() style, described at http://www.php.net/date.
```
Comment (dan-f) [inline analytics_dashboard/settings/base.py]: At first glance I was confused as to why we're referencing the `PHP` documentation here. Then I saw that [`django.utils.dateformat`](https://github.com/django/django/blob/master/django/utils/dateformat.py) itself references that same page!

Would you please update this comment to explain that the string is indented for use with the `django.utils.dateformat` module which itself uses the PHP date() style?

**Comment by dan-f** [comment]
Hi @louyihua. Thanks for the PR and for cleaning up this bit of code to be more consistent! I have one minor comment. Once you address it, I'll merge in your change. Thanks again!

**Comment by louyihua** [comment]
@dan-f I'm done with the comments.

File: analytics_dashboard/settings/base.py [hunk 3]
```diff
@@ -321,8 +321,12 @@
 # used to construct the shortcut link to view/edit a course in Studio
 CMS_COURSE_SHORTCUT_BASE_URL = None
 
-# Used to determine how dates are displayed in templates
+# Used to determine how dates and time are displayed in templates
+# The strings are indented for use with the django.utils.dateformat
```
Comment (dan-f) [inline analytics_dashboard/settings/base.py]: Sorry to make another nitpick, but did you mean "intended" rather than "indented"?

**Comment by louyihua** [comment]
@dan-f  Sorry for this spelling mistake.

**Comment by dan-f** [comment]
:+1: thank you for the contribution! I'm merging this now.

**Comment by dan-f** [comment]
If you're all done with the branch, feel free to delete it.

**Comment by sarina** [comment]
@dan-f you don't have to worry about branches on external forks :)

**Comment by dan-f** [comment]
Whoops! Forgot it was a fork. I've become used to our million-branches-on-shared-repo flow :stuck_out_tongue:

**Pull request merged**